changelog.json

```
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "3.3.5": [
+      "[Added] Syntax highlighting now supports .cc files - #17503. Thanks @DylanDevelops!",
+      "[Fixed] Long file paths are correctly truncated in the conflicts dialog - #17595",
+      "[Fixed] Screen readers announce group name when navigating through grouped lists - #17567"
```

Removing this line as it's a fix that belongs to grouped lists, only enabled in beta.
```suggestion
      "[Fixed] Long file paths are correctly truncated in the conflicts dialog - #17595"
```